rdr_service/services/consent/files.py

```diff
@@ -623,39 +624,31 @@ def _try_to_parse_signature(self, footer_text):
         return None
 
     def get_signature_on_file(self):
-        self.shift_params = (8, 10)
-        signature_str = self._try_to_parse_signature("Participant's Name (printed)")
-        if signature_str:
-            return signature_str
-
-        self.shift_params = (73, 75)
-        signature_str = self._try_to_parse_signature("Participant's Name (printed)")
-        if signature_str:
-            return signature_str
-
-        self.shift_params = (8, 10)
-        signature_str = self._try_to_parse_signature("'s Name (printed)")
-        if signature_str:
-            return signature_str
-
-        self.shift_params = (73, 75)
-        signature_str = self._try_to_parse_signature("'s Name (printed)")
-        if signature_str:
-            return signature_str
+        signature_offsets = [
+            (8, 10),
+            (73, 75)
+        ]
 
-        self.shift_params = (8, 10)
-        signature_str = self._try_to_parse_signature("Name (printed)")
-        if signature_str:
-            return signature_str
+        signature_label_strings = [
+            "Participant's Name (printed)",
+            "'s Name (printed)",
+            "Name (printed)",
+            'Nombre del Participant',
```

Comment:
@kenny-skaggs FYI: I was looking at a CE "missing signature" error report from this morning and the file in question (that had a printed name on the signature line) had the text "Nombre de Participant"  (instead of "del").  I'm assuming this list needs to contain all possible exact string matches?